test/Microsoft.Framework.Runtime.FunctionalTests/Utilities/AssertHelpers.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace Microsoft.Framework.Runtime.FunctionalTests.Utilities
+{
+    public static class AssertHelpers
+    {
+        public static void SortAndEqual<T>(IEnumerable<T> expect, IEnumerable<T> actual, IComparer<T> comparer)
+        {
+            var expectArray = expect.OrderBy(elem => elem, comparer).ToArray();
+            var actualArray = actual.OrderBy(elem => elem, comparer).ToArray();
+
+            var missing = new List<T>();
+            var extra = new List<T>();
+
+            int i = 0, j = 0;
+            while (i < expectArray.Length && j < actualArray.Length)
```

Comment:
It is slow, O(n^2) maybe. Mind is O(n).

But I guess performance is not important here ...